config/billing_rates.yml

```diff
@@ -285,6 +345,36 @@
 - { id: ab65600f-912c-45cb-b3fa-e26351b1f91c, resource_type: PostgresStandbyVCpu,         resource_family: standard-i8g,               location: us-east-2,      unit_price: 0.0024854760, billed_by: duration, active_from: 2025-09-15T00:00:00Z, byoc: false }
 - { id: 4af9014c-ee3f-45df-99bc-ec489600d846, resource_type: PostgresStandbyVCpu,         resource_family: standard-i8g,               location: ap-southeast-2, unit_price: 0.0029793850, billed_by: duration, active_from: 2025-09-15T00:00:00Z, byoc: false }
 - { id: 273aa797-bcb1-4d71-9151-ed4a1e7d2152, resource_type: PostgresStandbyVCpu,         resource_family: standard-i8g,               location: eu-west-1,      unit_price: 0.0027389480, billed_by: duration, active_from: 2025-09-15T00:00:00Z, byoc: false }
+- { id: 399e03bc-580f-473c-b7b5-0fba746c700d, resource_type: PostgresVCpu,                resource_family: standard-i8ge,              location: us-west-2,      unit_price: 0.0024854760, billed_by: duration, active_from: 2025-09-15T00:00:00Z, byoc: false }
```

Comment:
How did you calculated appropriate values for these billing rates in the case of `byoc: false` and `byoc: true`?